Multisig/Logic/Models/TokenRegistry/BlockchainTokenStore.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  BlockchainTokenStore.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct BlockchainTokenStore {
+
+    func token(address: Address) -> Token? {
+        let erc721 = ERC721(address)
```

Comment:
should it be enforced to be called on a global queue only?

Reply:
I'm reluctant to do that, as I think it'll make it more complicated without additional benefit (incl. testing)